log/log.go

```diff
@@ -26,6 +27,21 @@ const (
 	NoLevel Level = ""
 )
 
+var levelOrder = map[Level]int{
```

Comment:
should we maybe change to use iota ?
then we can have String() method on the enum etc ... 
it would make ordering also easier.

Reply:
Level are strings in order to match with env var setups etc... ordering is just needed for know when and when not to log.